CUDA_UPGRADE_GUIDE.MD

```diff
@@ -53,7 +73,7 @@ Build Magma for Linux. Our Linux CUDA jobs use conda, so we need to build magma-
 1. Follow this [PR 1368](https://github.com/pytorch/builder/pull/1368) for all steps in this section
```

Comment:
Looks good. Is it possible to replace old CUDA 11.5/11.7 PRs with their CUDA 12.4 equivalents. So that its all up to date with latest CUDA update ?

Reply:
OK, making a pass to change CUDA 11.5/11.7 to CUDA 12.4 related PRs.

Reply:
Done, please review again.